### PR TITLE
Adding resourceModification as a supertype of contentModification & metadataModification

### PR DESCRIPTION
**Pull request by escowles**
* clarifying descriptions to indicate which types apply to files/bitstreams, metadata, or both

Addresses https://jira.duraspace.org/browse/FCREPO-1475

### CODE REVIEW

**Comment by escowles** [comment]
@ajs6f @acoburn @awoods : does this make it clearer which types apply to bitstreams vs. metadata?

File: audit.rdf [hunk 1]
```diff
@@ -51,6 +52,19 @@
...
+  </owl:NamedIndividual>
+
+  <owl:NamedIndividual rdf:about="http://fedora.info/definitions/v4/audit#resourceModification">
+    <rdfs:label xml:lang="en">resource modification</rdfs:label>
+    <skos:prefLabel xml:lang="en">resource modification</skos:prefLabel>
+    <rdfs:comment xml:lang="en">A content file or metadata is modified.</rdfs:comment>
+    <skos:definition xml:lang="en">A content file or metadata is modified.</skos:definition>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation"/>
+    <skos:narrower rdf:resource="http://fedora.info/definitions/v4/audit#bitstreamModification"/>
```
Comment (awoods) [inline audit.rdf]: Should this be:
`<skos:narrower rdf:resource="http://fedora.info/definitions/v4/audit#contentModification"/>`
??

Comment (ajs6f) [inline audit.rdf]: I thought `resourceModification` was the top of the mods?

**Comment by ajs6f** [comment]
Just to be clear, we're explaining removal as something disjoint with modification? Of course I'm fine with that, but we might want to add some `owl:differentFrom` incantations to that effect. Some people think of removal as being a kind of modification. (This is just a matter of natural language usage-- I just want to be as clear as possible and maybe a bit more.)

**Comment by escowles** [comment]
@ajs6f yes, the intention is for removal to be when a node/bitstream is deleted, and modification when it is updated.  Neither OWL nor SKOS has great vocabulary for expressing that they are disjoint, but I can add owl:differentFrom assertions (or just a owl:AllDifferent block saying all four individuals are different).

**Comment by ajs6f** [comment]
Cool, we're on the same page. Yeah, OWL in particular wants to be real careful about disjointness because of the anti-UNA attitude that comes with a global field of work, but I think we're justified here. And `AllDifferent` is better.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/ontology/commit/c77d81bf45f032530d801f12792da35c94207a53

**Pull request closed without merging**